packages/react-components/react-slider/library/src/components/Slider/useSliderState.tsx

```diff
@@ -37,10 +65,14 @@ export const useSliderState_unstable = (state: SliderState, props: SliderProps)
     }
   });
 
+  const stepsPercent = step && step > 0 ? (step * 100) / (max - min) : 0;
+
   const rootVariables = {
     [sliderDirectionVar]: state.vertical ? '0deg' : dir === 'ltr' ? '90deg' : '270deg',
-    [sliderStepsPercentVar]: step && step > 0 ? `${(step * 100) / (max - min)}%` : '',
+    [sliderStepsPercentVar]: stepsPercent ? `${stepsPercent}%` : '',
     [sliderProgressVar]: `${valuePercent}%`,
+    [sliderProgressValueVar]: `${valuePercent / 100}`,
```

Comment:
why divide by 100? what does that 100 mean ? can we provide some comments to this logic or encapsulate to self explanatory constants ?

Reply:
To proceed with further calculations, we must convert an integer value in the range of `0 to 100` into a floating-point value within the range of `0 to 1`.

Reply:
lets document it

Reply:
Updated PR, this is no longer relevant